openedx/core/djangoapps/certificates/api.py

```diff
@@ -19,6 +27,66 @@ def _enabled_and_instructor_paced(course):
     return False
 
 
+def certificates_viewable_for_course(course):
+    """
+    Returns True if certificates are viewable for any student enrolled in the course, False otherwise.
+    """
+    if course.self_paced:
+        return True
+    if (
+        course.certificates_display_behavior in ('early_with_info', 'early_no_info')
+        or course.certificates_show_before_end
+    ):
+        return True
+    if (
+        course.certificate_available_date
+        and course.certificate_available_date <= datetime.now(UTC)
+    ):
+        return True
+    if (
+        course.certificate_available_date is None
+        and course.has_ended()
+    ):
+        return True
+    return False
+
+
+def enrollment_is_verified(student, course_id):
+    """
+    Returns True if the student has a verified certificate enrollment in the course, False otherwise.
+    """
+    enrollment_mode, __ = CourseEnrollment.enrollment_mode_for_user(student, course_id)
+    return enrollment_mode in VERIFIED_CERTS_MODES
+
+
+def enrollment_is_active(student, course_id):
+    return CourseEnrollment.enrollment_mode_for_user(student, course_id)[1]
```

Comment:
Will these 2 enrollment helpers also be moved out of this file with EDUCATOR-1318?  They are both generic to enrollments and seem out of place in a module for certificates.

Reply:
Ah, you're right, these really belong on the `CourseEnrollment` class.

Reply:
And in fact, they already exist there.